app/components/views/TicketsPage/MyTicketsTab/MyTicketsTab.jsx

```diff
@@ -112,6 +131,59 @@ const MyTickets = ({ toggleIsLegacy }) => {
     setSortOrderKey(type.value);
   };
 
+  const arraysEqual = (a, b) => {
```

Comment:
we use `lodash` which has many optimized utility functions - https://docs-lodash.com/v4/is-equal/ can be used to compare arrays(and bunch of other stuff).